lib/ansible/plugins/lookup/first_found.py

```diff
@@ -43,7 +43,7 @@
 - name: |
         include tasks only if files exist.  Note the use of query() to return
         a blank list for the loop if no files are found.
-  import_tasks: '{{ item }}'
+  include_tasks: '{{ item }}'
```

Comment:
Ideally, since `first_found` isn't really a loop, this entire example should be removed, or updated.

Instead the value should be passed directly to `include_tasks`, and eliminate the loop completely.